src/backend/login/usingPassport.js

```diff
@@ -1,36 +1,44 @@
-const passport = require('passport');
-const OneLoginStrategy = require('passport-openid').Strategy;
+const saml = require('passport-saml');
 require('../lib/config');
-
-/* Set environmental variables for these
-https://www.onelogin.com/ to create your account.
-Tutorial is here: https://developers.onelogin.com/quickstart/authentication/nodejs
-OIDC_BASE_URI=https://openid-connect.onelogin.com/oidc
-OIDC_CLIENT_ID=ddf97d-xxx-xxxx-e92920
-OIDC_CLIENT_SECRET=f021f63bxxxx71f2c6a90c9b
-OIDC_REDIRECT_URI=http://localhost:3000/oauth/callback
+const fs = require('fs');
+const path = require('path');
+const logger = require('../utils/logger');
+/*
+Tutorial is here: https://medium.com/disney-streaming/setup-a-single-sign-on-saml-test-environment-with-docker-and-nodejs-c53fc1a984c9
+SAML2_BASE_URI=saml-poc
+SAML2_CLIENT_ID=ddf97d-xxx-xxxx-e92920
+SAML2_CLIENT_SECRET=f021f63bxxxx71f2c6a90c9b
+SAML2_REDIRECT_URI=http://localhost:3000/oauth/callback
 */
+let cert = null;
+
+const { SAML2_BASE_URI, SAML2_REDIRECT_URI } = process.env;
 
-// Configure the OpenId Connect Strategy
-// with credentials obtained from OneLogin
-const { OIDC_BASE_URI, OIDC_CLIENT_ID, OIDC_CLIENT_SECRET, OIDC_REDIRECT_URI } = process.env;
+try {
+  cert = fs.readFileSync(path.resolve(process.cwd(), './certs/key.pem'), 'utf8');
+} catch (error) {
+  logger.error({ error }, 'Unable to load certs/key.pem');
```

Comment:
Just noticed on Windows, where we hit this error, you haven't `require()`'ed  the `logger` module.

Reply:
```
FAIL test/posts.test.js

  ● Test suite failed to run

    TypeError: logger.error is not a function

      18 |   cert = fs.readFileSync(path.resolve(process.cwd(), './certs/key.pem'), 'utf8');

      19 | } catch (error) {

    > 20 |   logger.error({ error }, 'Unable to load certs/key.pem');

         |          ^

      21 | }

      22 | 

      23 | const samlStrategy = new saml.Strategy(

      at Object.<anonymous> (src/backend/login/usingPassport.js:20:10)

      at Object.<anonymous> (src/backend/web/routes/login.js:6:39)

```